modules/enabling-log-console-plugin.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * logging/log_visualization/log-visualization-ocp-console.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-log-console-plugin_{context}"]
+= Enabling the {log-plug} after you have installed the {clo}
+
+You can enable the {log-plug} as part of the {clo} installation, but you can also retroactively enable the plug-in if you have already installed the {clo} with the plug-in disabled.
```

Comment:
s/plugin/plug-in
Same minimalism nit as before, you might be able to cut _retroactively_ and keep your intended meaning.